test/caption-stream-test.js

```diff
@@ -216,6 +216,48 @@
     }, 'parsed the third caption');
   });
 
+  test('backspaces are applied to non-displayed memory', function() {
+    var captions = [];
+    cea608Stream.on('data', function(caption) {
+      captions.push(caption);
+    });
+
+    // RCL, resume caption loading
+    cea608Stream.push({ ccData: 0x1420 });
+    // '01'
+    cea608Stream.push({
+      ccData: ('0'.charCodeAt(0) << 8) | '1'.charCodeAt(0)
```

Comment:
I do not see what additional information this:
```js
cea608Stream.push({
  ccData: BACKSPACE_CCDATA
});
```
provides over this:
```js
cea608Stream.push({
  ccData: 0x1421 // backspace
});
```
(in fact, it is a little more opaque than the latter example).

In my mind, one duty of the tests is to teach the interface to the code to someone approaching the project for the first time. Using constants over literals hides some important details of actual operation. Is your objection mostly aesthetic?